client/src/components/FeedbackForm.js

```diff
@@ -0,0 +1,238 @@
+import React from 'react';
+import { Grid, FormLabel, TextField, Typography, DialogTitle, DialogContentText, DialogContent } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles';
+import RubricBubble from './RubricBubble'
+
+const useStyles = makeStyles((theme) => ({
+    dialogTitleEmbed: {
+        fontSize:"1.25rem"
+    },
+    feedbackRadioContainer: {
+        display: "flex",
+    },
+    feedbackRadioButton: {
+        paddingLeft: "1.5rem",
+        paddingRight: "1.5rem",
+        paddingTop: ".5rem"
```

Comment:
here and a bunch of places here, i would suggest 
```suggestion
        paddingTop: "0.5rem"
```

so that it's explicit. There's nothing wrong here, but IMO, better to be explicit - modern browsers would probably read this fine but you never know :) up to you if you have other opinions!